src/main/java/pl/nogacz/snake/application/PauseGame.java

```diff
@@ -0,0 +1,40 @@
+package pl.nogacz.snake.application;
+import javafx.scene.control.Alert;
+import javafx.scene.control.ButtonType;
+import java.util.Optional;
+
+/**
+ * @author Dawid Nogacz on 19.05.2019
```

Comment:
Bu kodu Dawid diye birine yazdırmadıysan bu author bilgisini sil. Ayrıca zaten git historysinde kimin hangi satırı değiştirdiği bilgisi olduğu için böyle author ifadelerine lüzum yok. Ne kadar az metin o kadar iyi bizim için.

Reply:
Tamam hocam, kendim yazdığım için siliyorum ve bir şey eklemiyorum. David'in yazdığı author satırlarını silmeyeyim değil mi, sonuçta o da gereksiz değişikliğe girer?